packages/mc-scripts/src/utils/auth.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+
+const authenticator = async (email, password, mcApiUrl) => {
```

Comment:
As mentioned in the other comment, I would name this `getToken`.

Also, we might want to use names arguments

```suggestion
const getToken = async ({ email, password, mcApiUrl }) => {
```

Reply:
I renamed this `getSessionData` instead of `getToken`. Since we are returning the expiry and token.